spec/rails_helper.rb

```diff
@@ -16,6 +16,10 @@
 
 require 'rspec/rails'
 
+SimpleCov.start do
+  add_filter 'app/uploaders/application_uploader'
```

Comment:
First of all configuration of simplecov are located in the [`.simplecov`](https://github.com/rubygarage/boilerplate/blob/develop/.simplecov) file.
Also, I'm not sure that it's a good idea to filter file, maybe we can create a dummy class where will be included this uploader and we can test it, what do you think?

Reply:
Propose to remove it